Regression with 1.2 on Windows

1.1.1 worked fine, and this is what I get now:

```python
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
cq_editor\main_window.py:35: in __init__
    self.viewer = OCCViewer(self)
cq_editor\widgets\viewer.py:59: in __init__
    self.create_actions(self)
cq_editor\widgets\viewer.py:113: in create_actions
    {'View' : [QAction(qta.icon('fa.arrows-alt'),
C:\FreshMiniconda\envs\cqgui\lib\site-packages\qtawesome\__init__.py:234: in icon
    return _instance().icon(*names, **kwargs)
C:\FreshMiniconda\envs\cqgui\lib\site-packages\qtawesome\__init__.py:130: in _instance
    _resource['iconic'] = IconicFont(*_BUNDLED_FONTS)
C:\FreshMiniconda\envs\cqgui\lib\site-packages\qtawesome\iconic_font.py:326: in __init__
    self.load_font(*fargs)
C:\FreshMiniconda\envs\cqgui\lib\site-packages\qtawesome\iconic_font.py:362: in load_font
    directory = self._get_fonts_directory()
C:\FreshMiniconda\envs\cqgui\lib\site-packages\qtawesome\iconic_font.py:576: in _get_fonts_directory
    fonts_directory = self._install_fonts(fonts_directory)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
self = <qtawesome.iconic_font.IconicFont object at 0x0000002959FBC160>
fonts_directory = 'C:\\FreshMiniconda\\envs\\cqgui\\Lib\\site-packages\\qtawesome\\fonts'
    def _install_fonts(self, fonts_directory):
        """
        Copy the fonts to the user Fonts folder.
    
        Based on https://stackoverflow.com/a/41841088/15954282
        """
        # Try to get LOCALAPPDATA path
        local_appdata_dir = os.environ.get('LOCALAPPDATA', None)
        if not local_appdata_dir:
            return fonts_directory
    
        # Construct path to fonts from LOCALAPPDATA
        user_fonts_dir = os.path.join(
            local_appdata_dir, 'Microsoft', 'Windows', 'Fonts')
        os.makedirs(user_fonts_dir, exist_ok=True)
    
        # Setup bundled fonts on the LOCALAPPDATA fonts directory
        for root, __, files in os.walk(fonts_directory):
            for name in files:
                src_path = os.path.join(root, name)
                dst_path = os.path.join(
                    user_fonts_dir,
                    os.path.basename(src_path))
    
                # Check if font already exists and copy font
                if os.path.isfile(dst_path) and filecmp.cmp(src_path, dst_path):
                    continue
                shutil.copy(src_path, user_fonts_dir)
    
                # Further process the font file (`.ttf`)
                if os.path.splitext(name)[-1] == '.ttf':
                    # Load the font in the current session
                    if not gdi32.AddFontResourceW(dst_path):
                        os.remove(dst_path)
                        raise WindowsError(
                            f'AddFontResource failed to load "{src_path}"')
    
                    # Store the fontname/filename in the registry
                    filename = os.path.basename(dst_path)
                    fontname = os.path.splitext(filename)[0]
    
                    # Try to get the font's real name
                    cb = wintypes.DWORD()
                    if gdi32.GetFontResourceInfoW(
                            filename, ctypes.byref(cb), None, GFRI_DESCRIPTION):
                        buf = (ctypes.c_wchar * cb.value)()
                        if gdi32.GetFontResourceInfoW(
                                filename, ctypes.byref(cb), buf, GFRI_DESCRIPTION):
                            fontname = buf.value
                    is_truetype = wintypes.BOOL()
                    cb.value = ctypes.sizeof(is_truetype)
                    gdi32.GetFontResourceInfoW(
                        filename, ctypes.byref(cb), ctypes.byref(is_truetype),
                        GFRI_ISTRUETYPE)
                    if is_truetype:
                        fontname += ' (TrueType)'
>                   with winreg.OpenKey(winreg.HKEY_CURRENT_USER, FONTS_REG_PATH, 0,
                                        winreg.KEY_SET_VALUE) as key:
E                                       FileNotFoundError: [WinError 2] The system cannot find the file specified
C:\FreshMiniconda\envs\cqgui\lib\site-packages\qtawesome\iconic_font.py:635: FileNotFoundError
______________________ ERROR at setup of test_traceback _______________________
```

--- a/qtawesome/iconic_font.py
+++ b/qtawesome/iconic_font.py
@@ -636,7 +636,7 @@ class IconicFont(QObject):
                         with winreg.OpenKey(winreg.HKEY_CURRENT_USER, FONTS_REG_PATH, 0,
                                             winreg.KEY_SET_VALUE) as key:
                             winreg.SetValueEx(key, fontname, 0, winreg.REG_SZ, filename)
-                    except FileNotFoundError:
+                    except OSError:
                         # Needed to support older Windows version where
                         # font installation per user is not possible/related registry
                         # entry is not available

--- a/qtawesome/iconic_font.py
+++ b/qtawesome/iconic_font.py
@@ -632,8 +632,15 @@ class IconicFont(QObject):
                         GFRI_ISTRUETYPE)
                     if is_truetype:
                         fontname += ' (TrueType)'
-                    with winreg.OpenKey(winreg.HKEY_CURRENT_USER, FONTS_REG_PATH, 0,
-                                        winreg.KEY_SET_VALUE) as key:
-                        winreg.SetValueEx(key, fontname, 0, winreg.REG_SZ, filename)
+                    try:
+                        with winreg.OpenKey(winreg.HKEY_CURRENT_USER, FONTS_REG_PATH, 0,
+                                            winreg.KEY_SET_VALUE) as key:
+                            winreg.SetValueEx(key, fontname, 0, winreg.REG_SZ, filename)
+                    except FileNotFoundError:
+                        # Needed to support older Windows version where
+                        # font installation per user is not possible/related registry
+                        # entry is not available
+                        # See spyder-ide/qtawesome#214 
+                        return fonts_directory
 
         return user_fonts_dir

